map/generate_chart.cpp

```diff
@@ -0,0 +1,127 @@
+#include "map/generate_chart.hpp"
+
+//#include "coding/png_memory_encoder.hpp"
+
+#include "base/assert.hpp"
+
+#include "std/fstream.hpp"
+
+#include "3party/agg/agg_conv_stroke.h"
+#include "3party/agg/agg_path_storage.h"
+#include "3party/agg/agg_pixfmt_rgba.h"
+#include "3party/agg/agg_rasterizer_scanline_aa.h"
+#include "3party/agg/agg_renderer_primitives.h"
+#include "3party/agg/agg_renderer_scanline.h"
+#include "3party/agg/agg_scanline_p.h"
+
+namespace
+{
+template <class TColor, class TOrder>
```

Comment:
We concluded (@ygorshenin, @mpimenov, @syershov, @mgsergio) that these "T" prefixes are ugly. Join us and get rid of them!